<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1078</width>
    <height>711</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../res/logo1.ico</normaloff>../res/logo1.ico</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="page">
       <layout class="QGridLayout" name="gridLayout">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <property name="spacing">
         <number>0</number>
        </property>
        <item row="0" column="0">
         <widget class="QFrame" name="widgetState">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>70</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>70</height>
           </size>
          </property>
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <property name="lineWidth">
           <number>0</number>
          </property>
          <layout class="QGridLayout" name="gridLayout_4">
           <property name="leftMargin">
            <number>1</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>10</number>
           </property>
           <property name="bottomMargin">
            <number>1</number>
           </property>
           <property name="horizontalSpacing">
            <number>12</number>
           </property>
           <property name="verticalSpacing">
            <number>0</number>
           </property>
           <item row="0" column="9">
            <widget class="QGroupBox" name="groupBox_2">
             <property name="title">
              <string>status                                     </string>
             </property>
             <widget class="QLabel" name="labelUser">
              <property name="geometry">
               <rect>
                <x>20</x>
                <y>20</y>
                <width>53</width>
                <height>16</height>
               </rect>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
             <widget class="QLabel" name="labelNum">
              <property name="geometry">
               <rect>
                <x>120</x>
                <y>20</y>
                <width>53</width>
                <height>21</height>
               </rect>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
             <widget class="QPushButton" name="mangerlogin">
              <property name="geometry">
               <rect>
                <x>180</x>
                <y>10</y>
                <width>75</width>
                <height>23</height>
               </rect>
              </property>
              <property name="text">
               <string>管理员登录</string>
              </property>
             </widget>
            </widget>
           </item>
           <item row="0" column="8">
            <widget class="LifeControl" name="widgetLifeTime" native="true">
             <property name="minimumSize">
              <size>
               <width>200</width>
               <height>50</height>
              </size>
             </property>
            </widget>
           </item>
           <item row="0" column="3">
            <widget class="QLabel" name="label_status">
             <property name="text">
              <string>Status</string>
             </property>
            </widget>
           </item>
           <item row="0" column="5">
            <spacer name="horizontalSpacer_2">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="0" column="2">
            <widget class="QPushButton" name="btnStartWork">
             <property name="minimumSize">
              <size>
               <width>80</width>
               <height>40</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>40</height>
              </size>
             </property>
             <property name="font">
              <font>
               <pointsize>-1</pointsize>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">QPushButton 
{
    background-color:rgb(220,220,220); 
    color:rgb(0,0,200); 
    border:1px solid rgb(200,200,200); 
    font-size:18px; 
    border-radius:5px;
}

QPushButton:hover
{
    background-color:rgb(200,200,200); 
    color:rgb(255,255,255);
    border:1px solid rgb(200,200,200); 
    border-radius:5px;
}

QPushButton:pressed
{
    background-color: rgb(250,250,250); 
    color:rgb(200,200,200);
    border:1px solid rgb(200,200,200); 
    border-radius:5px;
}

QPushButton:disabled
{
    background-color: rgb(200,200,200); 
    color:rgb(240,240,240);
    border:1px solid rgb(200,200,200); 
    border-radius:5px;
}
</string>
             </property>
             <property name="text">
              <string>开始工作</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QPushButton" name="btnConnect">
             <property name="minimumSize">
              <size>
               <width>80</width>
               <height>40</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>40</height>
              </size>
             </property>
             <property name="font">
              <font>
               <pointsize>-1</pointsize>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">QPushButton 
{
    background-color:rgb(220,220,220); 
    color:rgb(0,0,200); 
    border:1px solid rgb(200,200,200); 
    font-size:18px; 
    border-radius:5px;
}

QPushButton:hover
{
    background-color:rgb(200,200,200); 
    color:rgb(255,255,255);
    border:1px solid rgb(200,200,200); 
    border-radius:5px;
}

QPushButton:pressed
{
    background-color: rgb(250,250,250); 
    color:rgb(200,200,200);
    border:1px solid rgb(200,200,200); 
    border-radius:5px;
}

QPushButton:disabled
{
    background-color: rgb(200,200,200); 
    color:rgb(240,240,240);
    border:1px solid rgb(200,200,200); 
    border-radius:5px;
}
</string>
             </property>
             <property name="text">
              <string>环境检测</string>
             </property>
            </widget>
           </item>
           <item row="0" column="6">
            <widget class="QLabel" name="labelWorkMode">
             <property name="minimumSize">
              <size>
               <width>120</width>
               <height>0</height>
              </size>
             </property>
             <property name="font">
              <font>
               <pointsize>15</pointsize>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>WorkMode</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="labelLogo">
             <property name="minimumSize">
              <size>
               <width>120</width>
               <height>0</height>
              </size>
             </property>
             <property name="font">
              <font>
               <pointsize>15</pointsize>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>Logo</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item row="0" column="7">
            <widget class="QLabel" name="labelStepCycle">
             <property name="font">
              <font>
               <family>Arial</family>
               <pointsize>16</pointsize>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">color:rgba(0, 60, 120);</string>
             </property>
             <property name="text">
              <string>0s</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QWidget" name="widget" native="true">
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <property name="spacing">
            <number>2</number>
           </property>
           <property name="leftMargin">
            <number>1</number>
           </property>
           <property name="topMargin">
            <number>1</number>
           </property>
           <property name="rightMargin">
            <number>1</number>
           </property>
           <property name="bottomMargin">
            <number>1</number>
           </property>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1078</width>
     <height>23</height>
    </rect>
   </property>
   <property name="defaultUp">
    <bool>false</bool>
   </property>
   <property name="nativeMenuBar">
    <bool>true</bool>
   </property>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>帮助</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="actionUserGuide"/>
   </widget>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>高级设置</string>
    </property>
    <addaction name="actionCfg"/>
    <addaction name="actionSetOrder"/>
    <addaction name="actionMES"/>
    <addaction name="separator"/>
    <addaction name="actionGuide"/>
    <addaction name="actionScrew"/>
    <addaction name="actionAlgorithmDebug"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>操作控制</string>
    </property>
    <addaction name="actionEnvCheck"/>
    <addaction name="separator"/>
    <addaction name="actionStartTest"/>
    <addaction name="actionStopTest"/>
    <addaction name="separator"/>
    <addaction name="actionExitSys"/>
   </widget>
   <widget class="QMenu" name="menu_4">
    <property name="title">
     <string>权限管理</string>
    </property>
    <addaction name="actionAuthoritySwitch"/>
    <addaction name="separator"/>
    <addaction name="actionPackagePrint"/>
   </widget>
   <addaction name="menu_3"/>
   <addaction name="menu"/>
   <addaction name="menu_4"/>
   <addaction name="menu_2"/>
  </widget>
  <action name="actionAbout">
   <property name="text">
    <string>关于</string>
   </property>
  </action>
  <action name="action">
   <property name="text">
    <string>参数设置</string>
   </property>
  </action>
  <action name="actionChangePlugin">
   <property name="text">
    <string>工序选择</string>
   </property>
  </action>
  <action name="actionCfg">
   <property name="text">
    <string>参数配置</string>
   </property>
   <property name="toolTip">
    <string>参数配置</string>
   </property>
  </action>
  <action name="actionSetOrder">
   <property name="text">
    <string>设置工单号</string>
   </property>
  </action>
  <action name="actionMES">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>MES调试</string>
   </property>
   <property name="toolTip">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Microsoft YaHei UI'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; color:#5500ff;&quot;&gt;MES调试&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </action>
  <action name="actionLogin">
   <property name="text">
    <string>重新登录</string>
   </property>
  </action>
  <action name="actionGuide">
   <property name="text">
    <string>引导提示配置</string>
   </property>
   <property name="toolTip">
    <string>引导提示配置</string>
   </property>
  </action>
  <action name="actionScrew">
   <property name="text">
    <string>电批配置</string>
   </property>
  </action>
  <action name="actionAlgorithmDebug">
   <property name="text">
    <string>算法调试</string>
   </property>
  </action>
  <action name="actionUserGuide">
   <property name="text">
    <string>使用说明</string>
   </property>
  </action>
  <action name="actionStopTest">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>停止测试</string>
   </property>
  </action>
  <action name="actionStartTest">
   <property name="text">
    <string>开始测试</string>
   </property>
  </action>
  <action name="actionExitSys">
   <property name="text">
    <string>退出程序</string>
   </property>
  </action>
  <action name="actionEnvCheck">
   <property name="text">
    <string>环境检测</string>
   </property>
  </action>
  <action name="actionClearLifeTime">
   <property name="text">
    <string>清除使用次数</string>
   </property>
  </action>
  <action name="actionSetMaxLifeTime">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>预设最大次数</string>
   </property>
  </action>
  <action name="actionAuthoritySwitch">
   <property name="text">
    <string>管理员登录</string>
   </property>
  </action>
  <action name="actionPackagePrint">
   <property name="text">
    <string>包装打印</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>LifeControl</class>
   <extends>QWidget</extends>
   <header>LifeControl.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
